src/main/java/com/teamfive/disscard/DissCardController.java

```diff
@@ -32,23 +35,28 @@ public class DissCardController {
 
     // ===== Fields =====
 
-    @Autowired
-    ICardService cardService;
+    private final ICardService cardService;
 
-    Logger log = LoggerFactory.getLogger(this.getClass());
+    private final Logger log = LoggerFactory.getLogger(this.getClass());
+
+    // Constructor injection for better testability and immutability
+    @Autowired
+    public DissCardController(ICardService cardService) {
+        this.cardService = cardService;
+    }
```

Comment:
Constructor injection for better testability.
Source: https://martinfowler.com/articles/microservices.html